nixos/modules/services/networking/hylafaxp.nix

```diff
@@ -0,0 +1,262 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  setupScript = ''
+    ## locks
+    mkdir -p -m 0755 /var/lock/uucp
+    chown "uucp:uucp" "/var/lock/uucp"
+    ## etc
+    cp -r ${pkgs.hylafaxp}/share/hylafax/etc/hylafax /etc
+    cat ${_pagesizes} - > /etc/hylafax/pagesizes
+    chown -R "uucp:uucp" "/etc/hylafax"
+    chmod -R 0755 /etc/hylafax
+    ## spool
+    cp -r ${pkgs.hylafaxp}/share/hylafax/var/spool/hylafax /var/spool
+    ${if (config.services.hylafaxp.config != "") then "cat ${_config} - > /var/spool/hylafax/etc/config" else ""}
+    ${if (config.services.hylafaxp.hfaxd.hosts != "") then "cat ${_hosts_hfaxd} - > /var/spool/hylafax/etc/hosts.hfaxd" else ""}
+    ${if (config.services.hylafaxp.faxDispatch != "") then "cat ${_FaxDispatch} - > /var/spool/hylafax/etc/FaxDispatch" else ""}
+    ${pkgs.ghostscript}/bin/gs -q -sDEVICE=tiffg3 -sFONTPATH="${pkgs.ghostscript}/share/ghostscript/fonts ${pkgs.ghostscript}/share/ghostscript/${pkgs.ghostscript.version}/Resource/Init ${pkgs.ghostscript}/share/ghostscript/${pkgs.ghostscript.version}/lib ${pkgs.ghostscript}/share/ghostscript/${pkgs.ghostscript.version}/Resource/Font ${pkgs.ghostscript}/share/ghostscript/fonts ${pkgs.ghostscript}/share/fonts/default/ghostscript ${pkgs.ghostscript}/share/fonts/default/Type1 ${pkgs.ghostscript}/share/fonts/default/TrueType" /var/spool/hylafax/bin/genfontmap.ps > /var/spool/hylafax/etc/Fontmap.HylaFAX 2>/dev/null
```

Comment:
We need ghostscript to produce font names and paths depending on what's available on the system and the specific ghostscript package version.